src/modules/Course/components/PageNavbar/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Breadcrumb, Tabs } from 'flowbite-react';
+import React from 'react';
+import { HiHome } from 'react-icons/hi';
+
+// interface PageHeaderProps {}
+
+interface PageHeaderTheme {
+  navbar: {
+    base: string;
+    item: {
+      base: string;
+    };
+  };
+}
+
+const PageHeader: React.FC = () => {
+  const theme: PageHeaderTheme = {
+    navbar: {
+      base: 'flex gap-2',
+      item: {
+        base: 'flex hover:bg-red-100 hover:text-red-600 rounded-lg items-center justify-center p-3 text-sm font-medium first:ml-0 disabled:cursor-not-allowed disabled:text-gray-400 disabled:dark:text-gray-500',
+      },
+    },
+  };
+
+  return (
+    <div className="my-8 space-y-8">
+      <div>
+        <Breadcrumb aria-label="Default breadcrumb example">
```

Comment:
The Breadcrumb could be separate and configurable. Maybe receiving a list of routes and compose the breadcrumb to display

Reply:
Yep, but not now. Let's first prepare the structure.